src/components/zen-avatar-icon/zen-avatar-icon.tsx

```diff
@@ -36,7 +36,7 @@ export class ZenAvatarIcon {
     if (this.initials) return this.initials;
 
     let initials = '';
-    if (this.userName) {
+    if (this.userName && this.userName.trim() != '') {
```

Comment:
```suggestion
    if (this.userName.trim()) {
```

Always `!==`... but in this case, you can use it as a truthy value.

Reply:
Done